### DISCUSSION TITLE
How do I update from 1.14.3

### DISCUSSION THREAD

**Discussion by FreshOuttaPrison** [dogecoin/dogecoin#2510] [Help needed]
I tried even full deleting and redownloading the core 64bit on windows and only downloads 1.14.3 version… what do I have to do to get the newest version? Could we get a simple update button added? As I’m only a common person I don’t know what I’m doing wrong. I’m sure this is a common issue with plenty of others?

**Comment by ReverseControl** [comment]
Here: https://github.com/dogecoin/dogecoin/releases/download/v1.14.4/dogecoin-1.14.4-win64.zip

This discussion has been had, see: https://github.com/dogecoin/dogecoin/issues/296
